src/generated/resources/assets/enderio/lang/zh_cn.json

```
@@ -0,0 +1,1109 @@
+  "gui.enderio.ioconfig.none": "禁用",
+  "gui.enderio.ioconfig.pull": "输入端",
+  "gui.enderio.ioconfig.push": "输出端",
+  "gui.enderio.nocap.desc": "插入电容\n机器才能工作！",
+  "gui.enderio.nocap.title": "缺少电容",
+  "gui.enderio.progress": "进度：%s%%",
+  "gui.enderio.range": "范围",
+  "gui.enderio.range.hide": "隐藏范围",
+  "gui.enderio.range.show": "显示范围",
+  "gui.enderio.redstone.active_with_signal": "收到信号工作",
+  "gui.enderio.redstone.active_without_signal": "收到信号不工作",
+  "gui.enderio.redstone.always_active": "始终工作",
+  "gui.enderio.redstone.mode": "红石模式",
+  "gui.enderio.redstone.never_active": "从不工作",
+  "gui.enderio.sag_mill_chance": "概率：%s%%",
+  "gui.enderio.sag_mill_chance_ball": "概率：%s%%（已受磨珠影响）",
+  "gui.enderio.spawner.disabled": "在配置中被禁用",
+  "gui.enderio.spawner.other_mod": "其他模组",
+  "gui.enderio.spawner.too_many_mob": "生物过多",
+  "gui.enderio.spawner.too_many_spawner": "刷怪笼过多",
+  "gui.enderio.spawner.unknown": "未知生物",
+  "guidebook.enderio.book_title": "书名",
+  "guidebook.enderio.landing_text": "文本",
+  "hint.enderio.connected_textures.text": "如果你想让电容库有连接纹理，请在客户端安装Athena模组",
+  "info.enderio.coordinate_selector.no_block": "可触及范围内没有方块",
+  "info.enderio.coordinate_selector.no_paper": "物品栏没有纸",
+  "info.enderio.darksteel.upgrade.activate": "右击以激活",
+  "info.enderio.darksteel.upgrade.available": "可用升级",
+  "info.enderio.darksteel.upgrade.cost": "花费等级：%s",
+  "info.enderio.darksteel.upgrade.direct": "存入",
+  "info.enderio.darksteel.upgrade.direct.description": "将获得的物品直接传送到物品栏",
+  "info.enderio.darksteel.upgrade.empowered.absorption": "%s%% 的耐久损耗被能量吸收",
+  "info.enderio.darksteel.upgrade.empowered.description": "给钢注入μI能量",
+  "info.enderio.darksteel.upgrade.empowered.efficiency": "充能后的效率 +%s",
+  "info.enderio.darksteel.upgrade.empowered.obsidian.efficiency": "挖掘黑曜石的效率 +%s",
+  "info.enderio.darksteel.upgrade.empowered.storage": "存储%sµI",
+  "info.enderio.darksteel.upgrade.empowered_l1": "注能",
+  "info.enderio.darksteel.upgrade.empowered_l2": "注能 II",
+  "info.enderio.darksteel.upgrade.empowered_l3": "注能 III",
+  "info.enderio.darksteel.upgrade.empowered_l4": "注能 IV",
+  "info.enderio.darksteel.upgrade.explosive.description": "让周围的泥土和岩石爆炸",
+  "info.enderio.darksteel.upgrade.explosive_l1": "爆炸 I",
+  "info.enderio.darksteel.upgrade.explosive_l2": "爆炸 II",
+  "info.enderio.darksteel.upgrade.explosive_penetration.description": "让被挖掘方块后面的泥土和岩石爆炸",
+  "info.enderio.darksteel.upgrade.explosive_penetration_l1": "穿透爆炸 I",
+  "info.enderio.darksteel.upgrade.explosive_penetration_l2": "穿透爆炸 II",
+  "info.enderio.darksteel.upgrade.fork": "叉子",
+  "info.enderio.darksteel.upgrade.fork.description": "有了叉子，谁还要锄？",
+  "info.enderio.darksteel.upgrade.no_xp": "经验不足",
+  "info.enderio.darksteel.upgrade.spoon": "勺子",
+  "info.enderio.darksteel.upgrade.spoon.description": "有了勺子，谁还要锹？",
+  "info.enderio.durability.amount": "耐久值：%s",
+  "info.enderio.energy.amount": "%sµI",
+  "info.enderio.headchance": "%s%%的概率掉落生物的头",
+  "info.enderio.too_many_levels": "你的经验已超21862级，太多了。",
```

```suggestion
  "info.enderio.too_many_levels": "你的经验等级已超21862级，经验值太多了。",
```
note 经验 include XP and Experience Level

---

the english text is not exactly using terms vanilla terms, so... if we use the formal Chinese terms, it can be a bit of wierd.